molecule/agent-install-test/molecule.yml

```diff
@@ -0,0 +1,88 @@
+---
+# This file is the central configuration entrypoint for Molecule. With this file, you can configure each tool that Molecule will employ when testing the role.
+dependency:
+  name: galaxy
+driver:
+  name: delegated
+  gcp_service_account_key: ${GOOGLE_APPLICATION_CREDENTIALS}
+  gcp_project_id: ${PROJECT}
+  zone: ${ZONE}
+  region: ${REGION}
+  ssh_user: ${SSH_USER}
+  ssh_pub_key_file: "${SSH_KEY_FILE}.pub"
+  ssh_key_file: ${SSH_KEY_FILE}
+  network_name: "ansible-network-${AGENT_TYPE}-${MOLECULE_SCENARIO_NAME}"
+  firewall_name: ansible-firewall
+  ip_cidr_range: 172.16.0.0/28
+lint: |
+  set -e
+  yamllint .
+  ansible-lint .
+platforms:
```

Comment:
[Optional] The internal testing framework has defined a set of platforms for Kokoro to pull. Is it possible to re-use that? Or does Ansible require platforms definition in its yaml?

Reply:
I believe Ansible/Molecule requires the platforms definition, however it may be possible to populate the platforms section with the platform arguments we pass to kokoro and then we could skip the create_* delete_* tasks. 

This would be useful, because at the moment theres no way I know of to limit platforms based on agent_type. ex. we can't include ubuntu-2010 as a platform because this will fail for the ops-agent.

Reply:
It appears dynamically populating platforms is difficult/impossible because molecule doesn't have all the capabilities that ansible does. https://github.com/ansible-community/molecule/issues/1183

One potential solution would be to dynamically create the entire molecule.yml file each time we run the kokoro tests. Another one would be creating more scenarios with the platforms hardcoded i.e. agent-install-test-{monitoring,logging,ops-agent}.

Reply:
I went with option 2 since that seems significantly easier. Also, after consolidating install/uninstall tests much of the redundancy of having agent-specific tests has been removed.